File: LatB/Python/Disk.py
class Disk():
    # Constructor
    def __init__(self):
        # Attribute-attribute untuk class Disk
        self.tipe = ""
        self.kapasitas = ""
        self.harga = 0
    # Get dan Set Attribute
    def setTipe(self, tipe):
        self.tipe = tipe
    def getTipe(self):
        return self.tipe
    def setKapasitas(self, kapasitas):
        self.kapasitas = kapasitas
    def getKapasitas(self):
        return self.kapasitas
    def setHarga(self, harga):
        self.harga=  harga
    def getHarga(self):
        return self.harga
    # Fungsi untuk menampilkan isi dari setiap attribute
    def printDisk(self):
        print("Disk Type: " + str(self.tipe))
        print("Disk Capacity: " + str(self.kapasitas))
        print("Disk Price: " + str(self.harga))

File: LatB/Php/Disk.php
<?php
class Disk{
    # attribute untuk class Disk
    private $tipe;
    private $kapasitas;
    private $harga;
    # Constructor untuk class Disk
    function __construct($tipe, $kapasitas, $harga)
    {
        $this->tipe = $tipe;
        $this->kapasitas = $kapasitas;
        $this->harga = $harga;
    }
    # Get dan Set setiap attribute private
    function setTipe($tipe){
        $this->tipe = $tipe;
    }
    function getTipe(){
        return $this->tipe;
    }
    function setKapasitas($kapasitas){
        $this->kapasitas = $kapasitas;
    }
    function getKapasitas(){
        return $this->kapasitas;
    }
    function setHarga($harga){
        $this->harga = $harga;
    }
    function getHarga(){
        return $this->harga;
    }
    # Fungsi untuk menampilkan isi dari semua attribute private di Disk
    function printDisk(){
        echo "Disk Type: " .$this->tipe."<br>";
        echo "Disk Capacity: " .$this->kapasitas."<br>";
        echo "Disk Price: " .$this->harga."<br>";
    }
    function __destruct()
    {
        
    }
}
?>